src/applications/disability-benefits/all-claims/content/incidentDate.jsx

```diff
@@ -3,34 +3,34 @@ import AdditionalInfo from '@department-of-veterans-affairs/formation/Additional
 
 import { getPtsdClassification } from './ptsdClassification';
 
-const sharedDescription = (
-  <p>
-    Please tell us when this event happened. If the event happened over a period
-    of time, please tell us around when the event started. (If you’re having
-    trouble remembering the exact date you can provide a month and year. You'll
-    have a chance to provide more information later.)
-  </p>
-);
-
-const eventHelpText = (
-  <AdditionalInfo triggerText="What if I can’t remember the date?">
+export const ptsdDateDescription = (
+  <div>
+    <h5>Event date</h5>
```

Comment:
Is this supposed to be an h5 or an h3? The flow seems inconsistent right now between this page, the medals page, and the Event Description page.